composeCore/src/main/java/com/hamdan/forzenbook/compose/core/composables/Composables.kt

```diff
@@ -252,3 +271,90 @@ fun ForzenbookDialog(title: String, body: String, buttonText: String, onDismiss:
         modifier = Modifier.padding(ForzenbookTheme.dimens.mediumPad_1)
     )
 }
+
+@Composable
+fun PillToggleSwitch(
+    imageLeft: Int,
+    leftDescription: Int,
+    imageRight: Int,
+    rightDescription: Int,
+    onToggle: () -> Unit,
```

Comment:
This should be the last argument so you can shortcut syntax it

Reply:
right